src/Microsoft.AspNetCore.Http.Connections.Client/ITransport.cs

```diff
@@ -2,11 +2,12 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Threading.Tasks;
 using System.IO.Pipelines;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Sockets.Client;
 
-namespace Microsoft.AspNetCore.Sockets.Client
+namespace Microsoft.AspNetCore.Http.Connections.Client
 {
     public interface ITransport
```

Comment:
We can do another pass moving more things to pubternal.

Reply:
:+1: